INTEGRATION_CHECKLIST.md

```
@@ -0,0 +1,316 @@
+
+```bash
+# Stop current backend
+# Then restart
+python fastapi_backend.py
+
+# Or if using Docker
+docker compose restart backend
+```
+
+You should see in the logs:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)